backend.native/tests/runtime/text/utf8.kt

```diff
@@ -70,51 +139,115 @@ fun <T: Any> checkThrows(e: KClass<T>, string: String, action: () -> Unit) {
     """.trimIndent())
 }
 
-fun checkUtf16to8Throws(string: String) = checkThrows(IllegalCharacterConversionException::class, string) {
-    string.toUtf8OrThrow()
+fun checkUtf16to8Throws(string: String) {
+    checkThrows(IllegalCharacterConversionException::class, string) { string.toUtf8OrThrow() }
+    checkThrows(CharacterCodingException::class, string) { string.encodeToByteArray(throwOnInvalidSequence = true) }
+}
+fun checkUtf16to8Throws(string: String, start: Int, size: Int) {
+    checkThrows(IllegalCharacterConversionException::class, string) { string.toUtf8OrThrow(start, size) }
+    checkThrows(CharacterCodingException::class, string) { string.encodeToByteArray(start, start + size, true) }
 }
 
-fun checkUtf16to8Throws(string: String, start: Int, size: Int) =
-        checkThrows(IllegalCharacterConversionException::class, string) {
-            string.toUtf8OrThrow(start, size)
-        }
 
-fun checkUtf8to16Throws(string: String, array: IntArray)
-        = checkThrows(IllegalCharacterConversionException::class, string) {
+// utils for checking malformed UTF-8 -> UTF-16 conversion
```

Comment:
// We start comments with capital letter and end with dot and avoid abbreviations.

Reply:
It looks like a comment for designating a group of functions, so it might follow the format:
`// ---------------------------- comment ----------------------------`

Reply:
+1